docs/docs/plugins-themes-and-starters.md

```diff
@@ -36,3 +36,72 @@ Gatsby offers [official starters](/docs/starters/#official-starters) for a defau
 - [Starter library](/starters/)
 - [Creating a starter](/docs/creating-a-starter/)
 - [Converting a starter to a theme](/docs/themes/converting-a-starter/)
+
+## Comparing differences
+
+Once you have a conceptual understanding of what a plugin, theme, and starter is, you may ask yourself what each is suited for in relation to each other.
+
+The following tables put plugins, themes, and starters side-by-side, showing what each is more appropriate for.
+
+**Legend**
+
+| Icon | Feature Capability    |
+| ---- | --------------------- |
+| ●    | Fully configurable    |
+| ◐    | Somewhat configurable |
+| ○    | Not configurable      |
+
+### Differences in maintainability
+
+When it comes to maintaining a Gatsby site, plugins and themes offer a distinct advantage to starters by being distributed as packages, making pulling in upstream changes in multiple Gatsby sites as easy as a new install.
+
+| Maintenance        | Plugin | Theme | Starter |
+| ------------------ | ------ | ----- | ------- |
+| Versioning         | ●      | ●     | ●       |
+| Install as Package | ●      | ●     | ○       |
+
+#### Versioning
+
+Starters can still be versioned inside a repository so that you could track issues or bugs associated with specific developments in the starter but it's more difficult to make updates with new changes.
+
+#### Installing as a package
+
+Because starters are "ejected" each time they are used, it's difficult to sync changes across multiple sites derived from the same starter. This problem was one of the motivating factors in developing the newer concept of themes for Gatsby. You can read more about why themes were developed in the [What are Gatsby Themes guide](/docs/themes/what-are-gatsby-themes/#gatsby-starters).
+
+### Differences in configurability
+
+Plugins and themes can expose a varying level of an API surface (meaning a piece of the plugin or theme that a user of it can interact with), constraining what is possible and providing sensible defaults to make them easier to start picking up and using.
+
+| Configuration         | Plugin | Theme | Starter |
+| --------------------- | ------ | ----- | ------- |
+| Shadowing             | ◐      | ●     | ○       |
+| Uses Multiple Plugins | ◐      | ●     | ●       |
+| Custom components     | ◐      | ●     | ●       |
+
+#### Shadowing
+
+The options exposed by a plugin or theme are up to the author. For example, a plugin or theme can allow a specific path to be provided in the `gatsby-config` so the plugin knows where to build pages from. In this case the user wouldn't be able adjust _how_ those pages are built, only from what path. As an escape hatch to change small pieces of an installed theme, [shadowing](/docs/themes/shadowing/) was introduced. Theme shadowing allows users to replace a file with their own version of it. Because themes are also plugins, shadowing is possible in plugins as well.
```

Comment:
Are all plugins shadowable or only some? My impressions was theme-ui is a bit unique?

Reply:
Yeah, here's the thread where I asked about this in Slack: https://gatsbyjs.slack.com/archives/CBQAWE7M2/p1580861936017300 

Really, I'm not aware of any difference between plugins and themes except for how they are conceptually understood.

Reply:
Gotcha, so I think this actually goes to the point below. Is there a way to do a section explicitly focused on the technical differences by convention. And then the other stuff is the limitations/uses.

Reply:
Do you mean adding another section (like h2 level) to summarize how they are different before doing comparisons? Would you suggest changing the headers? I guess I'm not clear on what structure isn't working and what would improve it.

Reply:
So this is just a potential option. But my current view is that some of the differences end up being repeated through out each section because they apply in all of those places. Mainly, a theme being a plugin and the main difference being convention of naming and usage. And then a starter being a snapshot in time that doesn't get updated. 

Those pieces of information are covered in other sections but my read is that they're important for framing everything. 

I think there are two options, though I'm open to other suggestions.
1. Make it another h2 before this comparing differences section called "Technical Intent and Associated Conventions", or something better than that.
2. Under the comparing differences section make the first h3 an overview of those pieces of information that help frame the other consideration. Something like "Overarching Considerations".

Let me know if that helps clarify things. And my apologies for not making the suggestions directly in the PR - given the high level architecture considerations I wasn't sure that would add any clarity.

Reply:
Great points! That's super helpful, and I understand why you can't make that a specific suggestion in the PR. I'll make some broader changes to that structure the way you've suggested 👍

Reply:
Okay, thanks @laurieontech I did a couple things:

- added an additional h2 that gave the context that plugins/themes are technically the same, but distinct by how they are intended to be used
- reworded the h3's to add in the word considerations
- removed (or moved up into overviews) redundant stuff about plugins being like themes, or starters being ejected

Let me know if this seems clear enough or if there is more that should be done!